src/plotman/archive.py

```diff
@@ -130,13 +140,41 @@ def archive(dir_cfg, all_jobs):
         (archdir, freespace) = sorted(available)[index]
 
     if not archdir:
-        return(False, 'No archive directories found with enough free space')
-    
+        return 'No archive directories found with enough free space'
+
+    arch_jobs = get_running_archive_jobs(dir_cfg['archive'])
+    if arch_jobs:
+        # Build status list
+        status_list = 'Cannot retrieve status.'
```

Comment:
Calling this `status_list` with type str seems misleading. Can you either make it a list, or rename?